### DISCUSSION TITLE
production mode vs development

### DISCUSSION THREAD

**Discussion by sitesurfer** [spatie/laravel-ray#60] [Q&A]
Before I whizz a bug in, just checking whether there's more needed to do in order to disable ray in production mode.
I have set up a (non local) host and the app works 100% ok, however, switching to production in the .env file achieves nothing. I am still receiving ray() updates.

I have checked the config is not overriding etc, should I check elsewhere?

**Comment by freekmurze** [comment]
Do open up an issue for this. If possible create a small app that we can use to reproduce the error on our side. Thanks!
